example_configs/zitadel.md

```diff
@@ -14,16 +15,17 @@ Replace every instance of `dc=example,dc=com` with your configured domain.
 * Bind Password: `<user password>`
 
 ### User binding
-* Userbase: `ou=people,dc=example,dc=com`
+* Userbase: `dn`
```

Comment:
I'm not sure I understand this one. What's the doc for this?

Reply:
Their [doc](https://zitadel.com/docs/guides/integrate/identity-providers/ldap#create-a-new-ldap-provider) isn't really helpful, or maybe just I misinterpreted their doc.

I took a look on lldap's logs last night, and notice that when `ou=people,dc=example,dc=com` is used, lldap reported an error related to `unknown attribute` and asked me to use `ignore user attribute`. So after some researching, I found `dn` is the correct one used here.

Reply:
Ah yeah, they do mention "dn"

Their vocabulary is all over the place, it's all kinds of wrong...

Thanks for doing the legwork!